README.md

```diff
@@ -3,27 +3,11 @@ Puppet Specifications
 
 This repository contains specifications for [the project Puppet][1], and related technologies.
 
-Puppet Language Specification
----
-The Puppet Programming Language Specification is a specification of the Puppet Programming
-Language. The first published release of this specification specifies the language version 4.
-
-The version 4.0.0 is the first version of the specification (this to make it harmonize
-with the 4.0.0 release of Puppet). From that point, the intention is to keep the
-same specification version even if the minor version of the implementation changes (i.e.
-for other reasons that the specification has changed). When a specification change is made,
-it *may* skip several numbers to again harmonize with the Puppet implementation version number.
-
-Until Puppet 4.0 was released, there was just the "current" and "future" implementations
-of the language. As time goes on it will be impossible to use only those words as their meaning
-is relative to a particular release of puppet (the future in 3.6 is not the same as the future in 3.7, and again not the same as current in 4.0) - hence the need for a separate version of the
-specification.
-
-The [Puppet Project][1] is the reference implementation of the specification. 
+The goal of this repository is that an independent implementation could follow this specification and fully support any Puppet Modules functional under an existing implementation.
 
 Semantic Versioning
 ---
-The specifications follows semantic versioning with the following semantics:
+The specifications contained in this repositoriy follows semantic versioning with the following semantics:
```

Comment:
```suggestion
The specifications contained in this repository follows semantic versioning with the following semantics:
```

Reply:
Fixed, good catch. I totally missed that with all the copy and pasting in that section.